lib/puppet/functions/azure_key_vault/secret.rb

```diff
@@ -13,18 +13,24 @@
     required_param 'String', :secret_name
     required_param 'Hash',   :api_versions_hash
     optional_param 'String', :secret_version
+    optional_param 'Hash',   :identity
   end
 
-  def secret(cache, vault_name, secret_name, api_versions_hash, secret_version = '')
+  def secret(cache, vault_name, secret_name, api_versions_hash, secret_version = '', identity = nil)
     Puppet.debug("vault_name: #{vault_name}")
     Puppet.debug("secret_name: #{secret_name}")
     Puppet.debug("secret_version: #{secret_version}")
     Puppet.debug("metadata_api_version: #{api_versions_hash['metadata_api_version']}")
     Puppet.debug("vault_api_version: #{api_versions_hash['vault_api_version']}")
+    if identity
+      Puppet.debug("azure_key_vault::secret: explicit identity specified: #{identity}")
```

Comment:
Can you remove the `azure_key_vault::secret:` prefix on the log statements to be consistent with the other statements in the module?